baal/modelwrapper.py

```diff
@@ -178,17 +179,21 @@ def train_and_test_on_datasets(
             _ = self.train_on_dataset(
                 train_dataset, optimizer, batch_size, 1, use_cuda, workers, collate_fn, regularizer
             )
-            te_loss = self.test_on_dataset(test_dataset, batch_size, use_cuda, workers, collate_fn)
-            hist.append(self.get_metrics())
-            if te_loss < best_loss:
-                best_epoch = e
-                best_loss = te_loss
-                if return_best_weights:
-                    best_weight = deepcopy(self.state_dict())
-
-            if patience is not None and (e - best_epoch) > patience and (e > min_epoch_for_es):
-                # Early stopping
-                break
+            if e%skip_epochs==0:
+                te_loss = self.test_on_dataset(test_dataset, batch_size, use_cuda, workers, collate_fn)
+                hist.append(self.get_metrics())
+                if te_loss < best_loss:
+                    best_epoch = e
+                    best_loss = te_loss
+                    if return_best_weights:
+                        best_weight = deepcopy(self.state_dict())
+
+                if patience is not None and (e - best_epoch) > patience and (e > min_epoch_for_es):
+                    # Early stopping
+                    break
+                hist.append(self.get_metrics())
```

Comment:
This is appending the metric twice I think. L184 already does it.

Reply:
Yes I made an mistake in the PR, sorry for that

Reply:
Hi @studentWUR, 
Thank you so much for your Pull Request. Would you mind pushing the requested change so that we can merge this branch?

Reply:
Sorry for my late reply; pushing a pull request is new for me; so I am slightly confused. In my forked repo I fixed the abovementioned typo, but how to push this to your git? Can you merge it from mine repo